Add NWB support for project schema, pre-commit and markdownlint (#42)
datajoint/element-lab: docs/src/tutorials.md

# Tutorials

## Installation

Installation of the Element requires an integrated development environment and database.
Instructions to setup each of the components can be found on the
[User Instructions](https://datajoint.com/docs/elements/user-guide/) page. 

## Element usage

> Element lab is a modular design of
> tables and dependencies that can be flexibly attached to any DataJoint workflow, but it is not a complete workflow by itself.  For example usages of  Element Lab and detailed tutorials, see the workflows listed below.

Reviewer: ```suggestion
Element Lab is a modular design of
tables and dependencies that can be flexibly attached to any DataJoint workflow, but it is not a complete workflow by itself.  For example usages of Element Lab and detailed tutorials, see the workflows listed below.
```

# Tutorials

> For detailed tutorials, please visit our documentation sites for the modality-specific
> Elements:
>   
>   - [Element Array Ephys](https://datajoint.com/docs/elements/element-array-ephys/latest/tutorials/)
>   - [Element Calcium Imaging](https://datajoint.com/docs/elements/element-calcium-imaging/latest/tutorials/)
>   - [Element DeepLabCut](https://datajoint.com/docs/elements/element-deeplabcut/latest/tutorials/)
>   - [List of all Elements](https://datajoint.com/docs/elements/)

Reviewer: ```suggestion
```

## Installation

Installation of the Element requires an integrated development environment and database.
Instructions to setup each of the components can be found on the
[User Instructions](https://datajoint.com/docs/elements/user-guide/) page. 

> ## Element usage
> 
> `element-lab` is not a complete workflow by itself, but rather a modular design of
> tables and dependencies that can be flexibly attached to any DataJoint workflow.
> 
> See the
> [workflow-calcium-imaging](https://github.com/datajoint/workflow-calcium-imaging) and
> [workflow-array-ephys](https://github.com/datajoint/workflow-array-ephys) repositories
> for example usages of `element-lab`.

Reviewer: ```suggestion
## Element usage

Element lab is a modular design of
tables and dependencies that can be flexibly attached to any DataJoint workflow, but it is not a complete workflow by itself.  For example usages of  Element Lab and detailed tutorials, see the workflows listed below.

  - Element Array Ephys - [Docs](https://datajoint.com/docs/elements/element-array-ephys/latest/tutorials/) & [Code repository](https://github.com/datajoint/workflow-array-ephys)
  
  - Element Calcium Imaging - [Docs](https://datajoint.com/docs/elements/element-calcium-imaging/latest/tutorials/) & [Code repository](https://github.com/datajoint/workflow-calcium-imaging)
  
  - Element DeepLabCut - [Docs](https://datajoint.com/docs/elements/element-deeplabcut/latest/tutorials/) & [Code repository](https://github.com/datajoint/workflow-deeplabcut)
```

# Tutorials

> Please visit our other Element documentation sites for detailed tutorials:

Reviewer: ```suggestion
For detailed tutorials, please visit our documentation sites for the modality-specific Elements:
```

  
  - [Element Array Ephys](https://datajoint.com/docs/elements/element-array-ephys/latest/tutorials/)
  - [Element Calcium Imaging](https://datajoint.com/docs/elements/element-calcium-imaging/latest/tutorials/)
  - [Element DeepLabCut](https://datajoint.com/docs/elements/element-deeplabcut/latest/tutorials/)

>   - [Full List](https://datajoint.com/docs/elements/)

Reviewer: ```suggestion
  - [List of all Elements](https://datajoint.com/docs/elements/)
```
